pkg/client/dynamic/clientset/tekton/tekton.go

```diff
@@ -28,16 +27,6 @@ import (
 // WithClient adds Tekton related clients to the Dynamic client.
 func WithClient(client dynamic.Interface) clientset.Option {
```

Comment:
I don't think we are using this function anywhere else apart from the places removed.

Reply:
@khrm seems to work!